src/Integration/LocalServices/IProjectSystemHelper.cs

```diff
@@ -84,10 +84,5 @@ public interface IProjectSystemHelper : IVsHierarchyLocator
         /// </summary>
         /// <returns>Project kinds GUIDs for the project</returns>
         IEnumerable<Guid> GetAggregateProjectKinds(IVsHierarchy hierarchy);
-
-        /// <summary>
-        /// Returns a flag indicating whether there is fully-opened solution or not.
-        /// </summary>
-        bool IsSolutionFullyOpened();
```

Comment:
Is there an implementation of the method in `ConfigurableVsProjectSystemHelper` that can be removed?